kebechet/config.py

```diff
@@ -84,15 +85,26 @@ def download_conf_from_url(url: str, service: str):
         return tempfile
 
     @classmethod
-    def run_url(cls, url: str, service: str):
+    def run_webhook(cls, payload: dict):
+        payload = PayloadParser(payload)
+        parsed_payload = payload.parsed_data()
+        if parsed_payload:
+            cls.run_url(parsed_payload['url'], parsed_payload['service_type'], True)
```

Comment:
The same as above - the program was asked to do a job.

Reply:
I will raise an exception if parsed_payload is None.

Reply:
Yes, correct. No other exception needed. :+1: Maybe the `if` statement is redundant then as `parsed_payload` always returns a non-empty dict.

Reply:
Let's have this for now, as in-case, the service-type doesn't get defined? Should be have a check in `parsed_data` to see if all the parameters got defined?